algebird-util/src/main/scala/com/twitter/algebird/util/UtilAlgebras.scala

```diff
@@ -52,11 +52,9 @@ object UtilAlgebras {
   implicit def futureMonoid[T: Monoid]: Monoid[Future[T]] = new ApplicativeMonoid[T, Future]
   implicit def futureGroup[T: Group]: Group[Future[T]] = new ApplicativeGroup[T, Future]
   implicit def futureRing[T: Ring]: Ring[Future[T]] = new ApplicativeRing[T, Future]
```

Comment:
in fact future is not even a group because `Throw/Failure` has no inverse, same with Try.